bytecode/cfg.py

```diff
@@ -53,6 +53,15 @@ def _compute_stack_size(block, size, maxsize):
     if block.seen or block.startsize >= size:
         return maxsize
 
+    def update_size(delta):
+        nonlocal size
```

Comment:
I must say I am not a big fan on nonlocal. It seems to me it makes the code harder to follow. Couldn't we simply return the new size and maxsize ?